src/screens/ManageTag/ManageTag.tsx

```diff
@@ -166,10 +166,10 @@ function ManageTag(): JSX.Element {
   const orgUserTagAncestors = orgUserTagAncestorsData?.getUserTagAncestors;
 
   const goToSubTags = (tagId: string): void => {
-    navigate(`/orgtags/${currentUrl}/orgtagSubTags/${tagId}`);
+    navigate(`/orgtags/${currentUrl}/subTags/${tagId}`);
   };
 
-  const handleClick = (tagId: string): void => {
+  const goToManageTag = (tagId: string): void => {
     navigate(`/orgtags/${currentUrl}/managetag/${tagId}`);
```

Comment:
`currentUrl` needs to be changed to `organizationID`? It's a bit confusing here. I think I missed this in the last PR.

Reply:
yes, that's right, thank you.